idc_collections/collex_metadata_utils.py

```diff
@@ -24,6 +24,201 @@
 logger = logging.getLogger('main_logger')
 
 
```

Comment:
These need to be removed in favor of using the versions already in the file; there's no functional difference between the API and the WebApp using BQ.